src/modules/task/internal/store/task-db.ts

```diff
@@ -0,0 +1,28 @@
+import { Schema, Types } from 'mongoose';
+
+export interface TaskDb {
```

Comment:
Please follow convention used in account module. Rename it to `TaskDB`

Reply:
@jjalan  sir, this was previously ```TaskDB``` but in previous review I was told to change it to ```taskDb``` but linting does not allow this naming convention and linting shows to use PascalCase, so I used this ```TaskDb```.